.github/actions/setup-server/action.yml

```diff
@@ -10,7 +10,7 @@ runs:
 
     - name: Set filename for the server's output logs
       shell: bash
-      run: echo "SERVER_LOGS_FILE=logs/server_logs_${{ matrix.python-version }}" >> $GITHUB_ENV
+      run: echo "SERVER_LOGS_FILE=logs/server_logs" >> $GITHUB_ENV
```

Comment:
we don't need the postfix anymore?

Reply:
nope, cus we're picking the python version at random, not looping over all 4 supported versions.